faststream/app.py

```diff
@@ -154,7 +153,13 @@ async def run(
         self,
         log_level: int = logging.INFO,
         run_extra_options: Optional[Dict[str, "SettingField"]] = None,
-        sleep_time: float = 0.1,
+        sleep_time: Annotated[
+            float,
+            deprecated(
+                "Deprecated in **FastStream 0.5.0**. "
```

Comment:
Should I set patch-version here? What version should I set?

Reply:
0.5.24